tools/simhostroute.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -x
+
+# Helper script to set up host route to NuttX simulator
+# and set up IP Tables to allow it to access the 
+# internet.
+#
+# Note that on Linux you may have to set kernel capabilities
+# on the nuttx executable to allow NuttX to access the tap device:
+#
+# sudo setcap cap_net_admin+ep ./nuttx
+
+if [ $# != 2 ]; then
+  echo "Usage: $0 <interface> <on|off>"
+  exit 1
+fi
+
+IF_HOST=$1
+STATUS=$2
+
+IF_BRIDGE=nuttx0
+IP_NET="10.0.1.0/24"
+IP_NETMASK="255.255.255.0"
+IP_BROADCAST="10.0.0.255"
+IP_HOST="10.0.1.1"
+IP_NUTTX="10.0.1.2"
+
+if [ "$STATUS" == "on" ]; then
+    ip link add $IF_BRIDGE type bridge
+    ifconfig $IF_BRIDGE $IP_HOST
+    ifconfig $IF_BRIDGE up
+    ifconfig -a
+    ip addr add $IP_HOST dev $IF_BRIDGE 
+    ifconfig $IF_BRIDGE netmask $IP_NETMASK
+    ip route delete $IP_NET
```

Comment:
don't need, done at line 47

Reply:
@xiaoxiang781216 I don't understand, will you elaborate?

Reply:
@xiaoxiang781216 I think I get it. I'll refactor this script to match the netusb.sh script (after fixing that one per your comments).

Reply:
I mean that "ip route delete $IP_NET" already do in the off path, so we don't need delete again in on path. For example, you don't delete $IP_NUTTX/32 in the on path.